eng/pipelines/templates/Apex_Tests_On_Windows.yml

```diff
@@ -85,8 +87,36 @@ stages:
 
               $(DartLab.Path)\Scripts\VisualStudio\Build\Get-BaselineBuildCommitIDs.ps1 -BuildJson $baselineBuildsFile -OutVariableName "BaselineBuildCommitIds"
               } catch {
-                Write-Host "##vso[task.LogIssue type=error;]Unable to get Baseline build commit ids: $_"
-              }            
+                Write-Host "Unable to get Baseline build commit ids: $_"
+              }
+        - task: PowerShell@2
+          name: SetVisualStudioBaseBuildID
+          displayName: Set 'VisualStudio.BaseBuild.ID'
+          retryCountOnTaskFailure: 3
```

Comment:
checked with @vishavpandhi offline. This parameter only enables retries for the following steps. Apex/End to end tests will not be rerun upon failure.

- Get Baseline build commit ids
- Set 'VisualStudio.BaseBuild.ID'
- Set 'VisualStudio.BaseBuild.ProductsDropName
- Set 'VisualStudio.BaseBuild.ProductsDropName' to drop from target build